jira-api/routes/api/v1/routes.js

```diff
@@ -79,31 +79,26 @@ app.get('/api/v1/getUsersForGroups/:groupName', asyncMiddleware(async function(r
     })
 }));
 
-app.get('/api/v1/getTimeForUsers/', asyncMiddleware(async function(req, res, next){
-
-    let group = req.query.groupName;
-    let startDate = req.query.startDate;
-    let endDate = req.query.endDate;
-    let boardId = req.query.boardId;
-    let sprintId = req.query.sprintId;
-
+app.get('/api/v1/getTimeForUsers/', asyncMiddleware(async (req, res, next) => {
```

Comment:
Use arrow functions :)  The only time I can really think of not using them is in event listener call backs so that ```this``` is bound to the event object, but really just use the event object in that case